Taskfile.cargo-tools.yml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://json.schemastore.org/taskfile.json
+
+version: 3
+
+tasks:
```

Comment:
No stress at all, but out of interest — any reason to split this out into a different file?

Reply:
Each time the `Taskfile.yml` is modified (even a mere modification of a comment, for example), the Docker cache is invalidated.
By copying files that are less likely to be modified, I believe we can make better use of the cache.

Reply:
OK. But the Taskfile doesn't change often — I would lean towards having the repo simpler even if it means running too many workflows. If we did this for everything we'd have many many tiny files and it would be difficult to work with the repo.

If we had a full build system like Bazel, then we could do this, but it's very approximate at best, and so somewhat fruitless to optimize it...